abs.yaml

```
@@ -0,0 +1,2 @@
+ build_env_vars:
```

A leading SPACE ??  You don't build for `py313`...whereon you discover you need `sphinx` for `py313` which requires `sphinxcontrib-serializing` (which wants `defusedxml` for testing...) which requires `sphinx` ... um?

```suggestion
build_env_vars:
```

but then you have to fix the above and figure out manual uploading.